app/views/my_facilities/bp_controlled.html.erb

```diff
@@ -168,6 +168,30 @@
             </tbody>
           </table>
         <% end %>
+        <div class="d-flex mt-4">
+          <em class="h-12px w-12px mr-8px bg-red br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-20%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-orange br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            21-36%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-yellow br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            37-50%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            &gt;50%
+          </p>
+        </div>
```

Comment:
The `.high-is-good` key doesn't match the CSS in `_dashboard.css`.
* `0%` is grey in CSS, but isn't listed as a key item
* `1-20%` should be red
* `21-35%` should be orange
* `36-49%` should be yellow
* `≥50%` should be green

Reply:
Done. Good catches.